config/types/files_test.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"github.com/vincent-petithory/dataurl"
+	"strings"
+	"testing"
+)
+
+func TestDataURL(t *testing.T) {
+	testString1 := "hello world"
+	dataUrl1 := makeDataUrl([]byte(testString1))
+	decodedUrl1, err := dataurl.DecodeString(dataUrl1)
+	if err != nil {
+		t.Fatalf("DecodeString #1 failed: %v", err)
+	}
+	resultStr1 := string(decodedUrl1.Data)
```

Comment:
```suggestion
	resultStr1 := string(decodedUrl1.Data)
	if strings.HasPrefix(dataUrl1, "data:;base64,") {
		t.Errorf("base64 encoding used instead of url encoding: %v", dataUrl1)
	}
```